spec/facades/yelp_facade_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe YelpService do
+  describe '::search_nearby' do
+    it 'returns array of business poros', :vcr do
+      manhattan = YelpFacade.search_nearby('manhattan', 'denver, co')
+
+      expect(manhattan).to be_an Array
+      expect(manhattan.first).to be_a Business
+    end
+  end
+end
```

Comment:
Testing here looks a little sparse? Not super worried for this PR but we need to test user auth for the request, as well as sad paths and all that.

Reply:
Prob can do what I did... initially just had a happy path, and then created a separate branch for sad paths since I had forgotten to do that too.